tests/clock/ClockUtils.py

```diff
@@ -0,0 +1,384 @@
+                logging.info(f'Timezone in timedatectl: "{timedatectl_timezone}"\nTimezone name: '
+                             f'"{timedatectl_tz_name}"\nTimezone abbreviation: "{timedatectl_tz_abbreviation}" ')
+
+            with allure.step(f'Verify timezone name "{timedatectl_tz_name}" from timedatectl is in valid timezones'):
+                logging.info(f'Verify timezone name "{timedatectl_tz_name}" from timedatectl is in valid timezones')
+                valid_timezones = ClockUtils.get_valid_timezones(duthosts)
+                pytest_assert(timedatectl_tz_name in valid_timezones,
+                              f'Error: string "{timezone_str}" is not in the valid timezones list')
+
+            with allure.step(f'Verify that the given timezone "{timezone_str}" equals to timezone abbreviation in '
+                             f'timedatectl "{timedatectl_tz_abbreviation}"'):
+                logging.info(f'Verify that the given timezone "{timezone_str}" equals to timezone abbreviation in '
+                             f'timedatectl "{timedatectl_tz_abbreviation}"')
+                ClockUtils.verify_value(expected=timedatectl_tz_abbreviation, actual=timezone_str)
+
+    @staticmethod
+    def verify_value(expected, actual, should_be_equal=True):
+        """
+        @summary:
+            Asserts a given value is as expected
+        @param expected: expected value
+        @param actual: actual given value
+        @param should_be_equal: whether the values should be equal or not
+        """
+        expected_to_print = "''" if expected == '' else expected
+        actual_to_print = "''" if actual == '' else actual
+
+        if should_be_equal:
+            with allure.step(f'Verify that actual value - {expected_to_print} is as expected - {actual_to_print}'):
+                logging.info(f'Verify that actual value - {expected_to_print} is as expected - {actual_to_print}')
+                pytest_assert(actual == expected, f'Error: Values are not equal.\n'
+                                                  f'Expected: {expected_to_print}\t{type(expected)}\n'
+                                                  f'Actual: {actual_to_print}\t{type(actual)}')
+        else:
+            with allure.step(f'Verify that actual value - {expected_to_print} '
+                             f'is different than expected - {actual_to_print}'):
+                logging.info(f'Verify that actual value - {expected_to_print} '
+                             f'is different than expected - {actual_to_print}')
+                pytest_assert(actual != expected, f'Error: Values are equal.\n'
+                                                  f'Expected: {expected_to_print}\t{type(expected)}\n'
+                                                  f'Actual: {actual_to_print}\t{type(actual)}')
+
+    @staticmethod
+    def verify_substring(expected_substr, whole_str):
```

Comment:
Why do we need a separate API to verify this? You can directly verify this in test flow. Unless you do special processing I don't recommend having a new API defined. This applies to all verify_ functions introduced here

Reply:
i removed these functions. will be noticed in the following commit tomorrow